express-post-json/index.js

```diff
@@ -0,0 +1,31 @@
+const express = require('express');
+const app = express();
+
+let nextId = 1;
+const grades = {};
+const array = [];
+function gradesInArray() {
+  for (const prop in grades) {
+    array.push(grades[prop]);
+  }
+  return array;
+}
+
+app.use(express.json());
+app.get('/api/grades', function (req, res) {
+  res.json(gradesInArray());
+
+});
```

Comment:
This is giving you a bigger and bigger list just by requesting the list again. You should only build the array inside of this route on-demand.